internal/exec/stages/files/filesystemEntries.go

```diff
@@ -512,3 +512,47 @@ func (s *stage) createEntries(entries []filesystemEntry) error {
 	}
 	return nil
 }
+
+// save the cex Volume keys
+func (s *stage) createCexVolumeKeys(config types.Config) error {
+	if len(config.Storage.Luks) == 0 {
+		return nil
+	}
+	s.Logger.PushPrefix("createCexVolumeKeys")
+	defer s.Logger.PopPrefix()
+	for _, luks := range config.Storage.Luks {
+		if luks.Cex.IsPresent() {
+			// zkey generates, validates and reencipher secure key. The information file
+			// which contains the description about keys for key management.
+			fileExt := []string{".info", ".skey"}
+			entries := []filesystemEntry{}
+			for _, zfile := range fileExt {
+				zkeyFile := filepath.Join(distro.LuksRealVolumeKeyFilePath(), "ignition-luks-"+luks.Name+zfile)
```

Comment:
That `ignition-luks-"+luks.Name+ext happens enough times in both this file and `luks.go` at this point that it probably makes sense to add tiny helpers somewhere that can be shared.

Reply:
Added helper function in `util` so that it can be shared across on `luks.go` and `filesystementries.go`.

Reply:
Done